app/src/components/submissionPage/SnapList.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import Snap from "../../firebase/snaps/Snap";
+
+interface SnapListProps {
+    snaps: Snap[];
+}
+
+const monthNames = [
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December",
+];
+
+const SnapList: React.FunctionComponent<SnapListProps> = ({ snaps }) => {
+    function formatAts(ats: string[]): string {
+        return ats.map((s) => `@${s}`).join(" ");
+    }
+
+    function formatTimestamp(date: Date) {
+        const day = date.getDate();
+        const month = monthNames[date.getMonth()];
+        const year = date.getFullYear();
+        const hrs = date.getHours();
+        const mins = date.getMinutes();
+        return `${day} ${month} ${year} at ${hrs}:${mins}`;
+    }
```

Comment:
JavaScript has very powerful date formatting tools that we should make use of. This has the advantage of taking into account a user's locale. I _think_ you'll want `toLocaleFormat` (see [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date)) and the cheatsheet for formatting strings [here](https://devhints.io/datetime).